# Exp2RM:Tissue-specific RNA methylation prediction from gene expression data using sparse regression models
**ABSTRACT**

N6-methyladenosine (m6A) is a highly prevalent and conserved post-transcriptional modification found in mRNA and long non-coding RNA (lncRNA), playing a vital role in diverse cellular processes. Identifying potential m6A sites within RNA sequences is crucial for understanding how the epitranscriptome influences biological processes. In this study, we introduce Exp2RM, a novel approach that constructs single site-based tissue-specific elastic net models for predicting tissue-specific methylation levels using gene expression data. In contrast to existing computational methods that heavily rely on RNA sequences for m6A site prediction, Exp2RM incorporates the relationship between tissue-specific gene expression levels and methylation sites. The resulting ensemble model demonstrate robust predictive performance for tissue-specific methylation levels, with an average R-squared value of 0.490 and a median R-squared value of 0.486 across all 22 human tissues. Additionally, our findings confirm the heterogeneity of methylation distribution across different tissues. Notably, training the models using data from tissues with similar methylation distributions significantly improves prediction accuracy, as indicated by the increase in the median R-squared value from 0.486 to 0.727. The results underscore the significance of tissue-specific methylation distribution in enhancing prediction performance. We also conducted a functional analysis of the genes associated with methylation sites in the model, revealing that Exp2RM accurately captures the functional roles of these coefficient genes in various biological processes. 


![Figure 1](https://github.com/jjnuo/Exp2RM/assets/57382360/2ef21bd4-96bf-4e5e-88f6-e411e0fd89ed)
